### ISSUE TITLE
permalinkAttrs is not working

### ISSUE THREAD

**Issue by nil** [valeriangalliat/markdown-it-anchor#65]
I'm tring to add custom attribute to the anchor link but it is not working. This is the code I'm using:

```js
import markdownIt from 'markdown-it';
import markdownItAnchor from 'markdown-it-anchor';

const md = new markdownIt().use(markdownItAnchor, {
  permalinkAttrs: () => ({ "aria-hidden":  false })
});
```
I've cloned this repository and when I run the tests with `yarn test` it returns an error.

**Comment by valeriangalliat** [comment]
Looks like running `npm run build` fixed the tests, I've published 5.2.7 with the new build!

Crazy how I didn't know myself that this project had a build system :joy:

**Comment by nil** [comment]
Yes, it is fixed 💯
